buildchain/buildchain/targets/local_image.py

```diff
@@ -169,6 +170,7 @@ def task(self) -> types.TaskDict:
             'title': lambda _: self.show('IMG BUILD'),
             'doc': 'Build {} container image.'.format(self.name),
             'actions': self._build_actions(),
+            'task_dep': ['check_for:skopeo', 'check_for:docker'],
```

Comment:
No need for `check_for:docker` + you shouldn't assign `task_dep` directly: if the user specified its own `task_dep` you'll overwrite them.

I recommend to append to `task_dep`, in the `__init__`:

```diff
@@ -107,6 +107,7 @@ class LocalImage(image.ContainerImage):
         self._save = save_on_disk
         self._build_args = build_args or {}
         kwargs.setdefault('file_dep', []).append(self.dockerfile)
+        kwargs.setdefault('task_dep', []).append('check_for:skopeo')
         super().__init__(
             name=name, version=version, destination=destination, **kwargs
         )
```